### ISSUE TITLE
lang.js not found because of git-submodules.

### ISSUE THREAD

**Issue by ghost** [rmariuzzo/Laravel-JS-Localization#42]
lang.js is not exist.

how can i?

**Comment by a-komarev** [comment]
Same for me... I've [downloaded it from submodule](https://github.com/rmariuzzo/Lang.js) manually and placed in Lang.js in package folder. But it's not production way.

**Comment by elminson** [comment]
The path of lang.js is \vendor\mariuzzo\laravel-js-localization\src\js is the same file generate with `php artisan lang:js` but with the difference Lang.setMessages is set with some translations.  I add `    <script type="text/javascript" src="messages.js"></script>` just for test purpose only using console.log

**Comment by fortranlee** [comment]
The Lang.js submodule is not initialized. What did I miss?
![qq 20160310101537](https://cloud.githubusercontent.com/assets/5966046/13657285/0494b134-e6a9-11e5-9d62-7d222e1e1708.png)

**Comment by dieg0v** [comment]
Same problem here... I temporarily fix this installing Lang.js with bower and then copy to vendor folder with composer:

bower.json: 

```
{
  "dependencies": {
    ...
    ...
    "lang.js": "~1.0.0"
  }
}
```

composer.json:

```
"post-update-cmd": [
   "php artisan clear-compiled",
   "php artisan optimize",
   "cp -R web/static/vendor/lang.js/ vendor/mariuzzo/laravel-js-localization/Lang.js/"
]
```

my .bowerrc:
```
{
  "directory" : "web/static/vendor"
}
```

Hope this help.

**Comment by hpolonkoev** [comment]
Maybe add an additional step after `composer update` into readme file.
`git submodule init`
`git submodule update`

Source from : http://stackoverflow.com/a/3796947

**Comment by rmariuzzo** [comment]
Seeing all this message, makes me think to revert the idea of using git-submodules. I will do some research before deciding. Any input will be highly appreciated.

**Comment by rmariuzzo** [comment]
This will be released in [v1.3.0](https://github.com/rmariuzzo/Laravel-JS-Localization/milestone/1).

**Comment by rmariuzzo** [comment]
@a-komarev, @hpolonkoev I was playing with Composer scripts and just noticed this:

> Only scripts defined in the root package's composer.json are executed. If a dependency of the root package specifies its own scripts, Composer does not execute those additional scripts.

Unfortunately, composer script is not a production option. I still want for the user to be able to just do: `composer require mariuzzo/laravel-js-localization:x.y.z` and done.

I'm thinking that I will manually put the `lang.js` file manually into the `lib/` directory and done. Just for the sake to release the 1.3.0 version then we can figure out a better approach.
